### ISSUE TITLE
Missing [U+26A1] ⚡

### ISSUE THREAD

**Issue by jeremyswann** [source-foundry/Hack#438]
I'm using [oh-my-posh](https://github.com/JanDeDobbeleer/oh-my-posh) on W10 PowerShell Core. Used a registry mod to change the console font to Hack. Unfortunately the cute little lightning bolt from power-line isn't a prat of the font set. Any chance you could add it please 🙏

**Comment by nsemrau** [comment]
I second that request. I use `wttr.in` (a weather forecast) in my terminal and they use that glyph for forecasting thunderstorms. The glyph is automatically substituted by another font on my setup but that breaks the ASCII table:

![thunder](https://user-images.githubusercontent.com/4363876/41255717-c303a8f8-6dc7-11e8-844c-5a5c4cf9d796.png)

**Comment by chrissimpkins** [comment]
Thanks for the report!  Will look into it and see what we can do.  Do you have any examples of typefaces that include this glyph?

**Comment by chrissimpkins** [comment]
@nsemrau Nicolas, that does not appear to be Hack in your renders.

**Comment by nsemrau** [comment]
You're right, sorry! I checked this with various fonts yesterday and still had the Deja Vu font set. I re-checked it now and the table looks similar with what is definitely Hack (the `1` gives it away :)).

**Comment by jeremyswann** [comment]
Hi @chrissimpkins, Google's [Nato Emoji](https://github.com/googlei18n/noto-emoji/blob/master/fonts/NotoEmoji-Regular.ttf) font has a good list of the weather set. Windows Segoe UI Emoji font comes built in to Win10, and replaces emojis inside of the browser and editors, but not in powershell yet.

**Comment by chrissimpkins** [comment]
Thank you! Will check it out.
